src/gateway/handlers/mod.ts

```diff
@@ -452,6 +452,9 @@ export type ClientEvents = {
   commandUsed: [ctx: CommandContext]
   commandError: [ctx: CommandContext, err: Error]
   commandNotFound: [msg: Message, parsedCmd: ParsedCommand]
+  commandUserBlacklisted: [ctx: CommandContext]
```

Comment:
just my thoughts but maybe blacklist'ed' is kinda overkill

Reply:
Do you have any suggestions for different wording? I just went with this as it's what came to mind first.

Reply:
Hmmm... 'commandBlockedUser' seems better than 'blacklist'